### PR TITLE
Add a specific mixin environment for doing CAPI mgmt clusters

### PR DESCRIPTION
**Pull request by mkjpryor**

### CODE REVIEW

File: environments/capi-mgmt/inventory/group_vars/all.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+community_images:
+  kube_1_26_5:
+    name: ubuntu-focal-kube-v1.26.5-230602-1757
+    source_url: "{{ community_images_base_url }}/ubuntu-focal-kube-v1.26.5-230602-1757.qcow2"
+    source_disk_format: qcow2
+    container_format: bare
+community_images_default_visibility: private
+community_images_update_existing_visibility: false
+
+capi_cluster_kubernetes_version: "1.26.5"
```
Comment (JohnGarbutt) [inline environments/capi-mgmt/inventory/group_vars/all.yml:45]: Is this out of date now?

**Review by JohnGarbutt — CHANGES_REQUESTED** [review]
Can we add a README or docs we can reference from the StackHPC kayobe config? Maybe that is overkill.

**Comment by mkjpryor** [comment]
> Can we add a README or docs we can reference from the StackHPC kayobe config? Maybe that is overkill.

Personally, I would rather we do the opposite, and keep the azimuth-config docs clean for Azimuth deployments.

**Review by scrungus — APPROVED** [review]
LGTM

**Pull request merged**